controllers/kokumetricsconfig_controller.go

```diff
@@ -436,6 +436,10 @@ func packageFiles(p *packaging.FilePackager, cr *metricscfgv1beta1.MetricsConfig
 		return
 	}
 
+	forcePackageFiles(p, cr)
+}
+
+func forcePackageFiles(p *packaging.FilePackager, cr *metricscfgv1beta1.MetricsConfig) {
```

Comment:
This is great separation. Distinct functions that do specific things are much better than passing a toggle that changes behavior.

Not a blocker, just a quibble over the name: this new function could be `packageFiles` since that is exactly what it does. The original could be `packageFilesDuringCycle` or something that better communicates that it only packages files when needed.

I think `maybePackageFiles` would be accurate but probably not the best name. 😆

Reply:
I can get behind changing the names. Lemme push up a commit real quick.